packages/mgt-chat/src/components/ChatList/ChatList.tsx

```diff
@@ -106,56 +105,37 @@ export const ChatList = ({
   ...props
 }: MgtTemplateProps & IChatListProps & IChatListMenuItemsProps) => {
   const styles = useStyles();
-
+  const chatListClient = useMemo(() => new StatefulGraphChatListClient(), []);
+  const chatListActions = useMemo(() => {
```

Comment:
Is there a reason on why you used useMemo instead of useCallback?

Reply:
Because I don't expect this declaration to ever change. For the others, I expect `ChatThreadsPerPage`, `SelectedChatId` to change at runtime.

Reply:
But you do have [chatListClient] as a dependency. useMemo and useCallback are going to do the same thing. I think the only difference is when it is calculated (useMemo immediately or after a dependency change; and useCallback when the function is called).

Reply:
If we look at what they did on Chat it was:
```
const [chatClient] = useState<StatefulGraphChatClient>(() => new StatefulGraphChatClient());
```

I don't know why we would deviate from what they did without a reason.